assets/src/edit-story/app/media/media3p/test/useFetchCategoriesEffect.js

```diff
@@ -34,6 +34,15 @@ jest.mock('../api', () => ({
   }),
 }));
 
+const mockShowSnackbar = jest.fn();
+jest.mock('../../../snackbar', () => ({
+  useSnackbar: () => {
```

Comment:
This can be:

useSnackbar: () => ({ showSnackbar: mockShowSnackbar }),

Reply:
Done